NuGet.config

```diff
@@ -1,8 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <packageSources>
-    <add key="AspNetVNext" value="https://www.myget.org/F/aspnetcidev/api/v3/index.json" />
-    <add key="CoreFx" value="https://dotnet.myget.org/f/dotnet-core/api/v3/index.json" />
+    <add key="AspNetCore" value="https://dotnet.myget.org/F/aspnetcore-ci-dev/api/v3/index.json" />
```

Comment:
Does it build if you remove this now? Maybe it'd be nice that Libuv doesn't depend on AspNet's artifacts at all.